### DISCUSSION TITLE
为什么 Clash 的配置的 ip 写死 172.16.0.2🤔

### DISCUSSION THREAD

**Discussion by CorrectRoadH** [vvbbnn00/WARP-Clash-API#85] [General]
好奇为什么 Clash 的配置的 ip 写死 172.16.0.2🤔
![CleanShot 2024-01-28 at 20 15 33@2x](https://github.com/vvbbnn00/WARP-Clash-API/assets/29306285/0f7ca237-0fd7-4747-a8ee-839581c4fe32)

是bug吗? 还是就是这样写的

**Comment by vvbbnn00** [comment]
你好，我是按照cloudflare的zerotrust手册中的默认IP设定的。

![image](https://github.com/vvbbnn00/WARP-Clash-API/assets/46409975/af12daa3-62aa-4ea5-bff4-416557f8b419)

是在使用过程中出现什么问题了吗？

**Comment by CorrectRoadH** [comment]
> 你好，我是按照cloudflare的zerotrust手册中的默认IP设定的。
> 
> ![image](https://private-user-images.githubusercontent.com/46409975/300262625-af12daa3-62aa-4ea5-bff4-416557f8b419.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDY0NDU5NTcsIm5iZiI6MTcwNjQ0NTY1NywicGF0aCI6Ii80NjQwOTk3NS8zMDAyNjI2MjUtYWYxMmRhYTMtNjJhYS00ZWE1LWJmZjQtNDE2NTU3ZjhiNDE5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTI4VDEyNDA1N1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTBkMDMxOWM2ZTJlYTQ5MWU3ZTJmMmYzMzkwNzNmZDgzMWU0MmYzYmZhYTAyOGNiY2U3YjkyMWM1NDYzNWIyM2EmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.8sMBAGT4EQvr_w2VPoyvO0Zny4JhBv6VBqygOnNKOJ0)
> 
> 是在使用过程中出现什么问题了吗？

没有遇到由此产生的bug，我只是单纯的不了解而已，好奇的问一下🥺

**Comment by vvbbnn00** [comment]
哦哦好的，我还以为哪里又出问题了😂
